common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -570,6 +569,20 @@
                                 gettext('Show posts by {username}.'), {username: username}
                             );
                             self.addSearchAlert(message, 'search-by-user');
+                            DiscussionUtil.safeAjax({
+                                url: DiscussionUtil.urlFor('followed_threads', response.users[0].id),
```

Comment:
Yeah, that will add followed thread in search result. If we need to add this thread we can add this code into `searchFor` without changing order of execution of these methods.
What are your thoughts on this?